src/clients/inst_autoinit.rb

```diff
@@ -150,6 +150,12 @@ def processProfile
           if newURI == ""
             return :abort
           else
+            # Updating new URI in /etc/install.inf (bnc#963487)
+            WFM.Execute(
+              path(".local.bash"),
+              "sed -i \'/AutoYaST:/c\\AutoYaST: #{newURI}\' /etc/install.inf"
```

Comment:
I think you don't need to escape the single quotes here, right? On the other hand, I didn't know about the `'c\\'` trick, nice :)